README.md

```diff
@@ -1,27 +1,35 @@
 <img src="./docs/img/openshift.png" width="180" align="right">
 
-# Odo - Developer Focused CLI for OpenShift
+# Odo - Developer focused CLI for OpenShift
 
 [![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo) [![CircleCI](https://circleci.com/gh/redhat-developer/odo/tree/master.svg?style=svg)](https://circleci.com/gh/redhat-developer/odo/tree/master) [![mattermost](/docs/img/mattermost.svg)](https://chat.openshift.io/developers/channels/odo)
 
-A fast iterative tool for deploying your source code straight to OpenShift.
+OpenShift Do (odo) is a  fast iterative CLI tool for developers who write, build, and deploy applications on OpenShift.
+Existing tools such as `oc` are more operations-focused and require a deep-understanding of Kubernetes and OpenShift concepts.
 
-## Features
+odo is  an opinionated CLI tool that supports fast, iterative development and that abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to focus on what's most important to them: code.
+
+odo improves the developer experience with OpenShift. It is designed to be simple and concise so that you can focus on coding rather than how to deploy your application. odo builds and deploys your code to your cluster as soon as you save your changes, giving you instant feedback to validate your changes in real-time. odo's syntax and design is centered around concepts already familiar to developers, such as: project, application and component.
+
+
+## Key features
```

Comment:
"features" should be capitalized.

Reply:
As per CCS documentation style guide and recommendations we keep headings in sentence case.